libraries/AP_BattMonitor/AP_BattMonitor_Scripting.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include "AP_BattMonitor_Backend.h"
+
+#if HAL_BATTMON_SCRIPTING_ENABLED
+
+#define AP_BATTMONITOR_SCRIPTING_TIMEOUT_MICROS (5000000) // backend becomes unhealthy script does not call handler for 5 seconds
+
+class AP_BattMonitor_Scripting : public AP_BattMonitor_Backend
+{
+public:
+    // Constructor
+    AP_BattMonitor_Scripting(AP_BattMonitor &mon,
+                          AP_BattMonitor::BattMonitor_State &mon_state,
+                          AP_BattMonitor_Params &params);
```

Comment:
```suggestion
    // Inherit constructor
    using AP_BattMonitor_Backend::AP_BattMonitor_Backend;
```
You can remove from the cpp also if this works.